lib/devise.rb

```diff
@@ -304,6 +304,10 @@ def sign_out_via=(sign_out_via)
   mattr_accessor :clean_up_csrf_token_on_authentication
   @@clean_up_csrf_token_on_authentication = true
 
+  # When false, Devise will not attempt to reload routes on eager load
+  mattr_accessor :reload_routes
+  @@reload_routes = true
```

Comment:
We don't need to set it on the generated config file, but we can add a commented example explaining the purpose of the setting.